lib/generator.js

```diff
@@ -165,6 +165,17 @@ const generateParameters = (parameters, customTypes, usedTypes) => {
   if (!parameters || !parameters.length) return buf;
 
   for (const parameter of parameters) {
+    if (
+      parameter.offset < CODE_PARAMS_OFFSET ||
+      parameter.offset % CODE_PARAMS_OFFSET !== 0
+    ) {
+      console.error(
+        `Warning: following parameter is incorrect and will not be generated:`,
+        parameter
```

Comment:
We should point to something here (i.e. filename, function name etc.), it'd be hard to find this parameter just by name.